### ISSUE TITLE
Warning in calling log_softmax

### ISSUE THREAD

**Issue by jinserk** [SeanNaren/deepspeech.pytorch#161]
I've updated pytorch from latest source repo, and met the following warning when I do a prediction.
```
model.py:44: UserWarning: Implicit dimension choice for log_softmax has been deprecated. Change the call to include dim=X as an argument.
  return torch.stack([F.log_softmax(input_[i]) for i in range(batch_size)], 0)
```
I've given F.log_softmax(input_[i], dim=2), but it led to an error. What's been changed?

**Comment by SeanNaren** [comment]
Fixed in #163

**Comment by ryanleary** [comment]
Heads up. That `dim` parameter was added recently. `deepspeech.pytorch` now requires an unreleased version of pytorch (> 0.2.0) to function.

**Comment by SarthakYadav** [comment]
I ran into the "log_softmax() got an unexpected keyword argument 'dim'" as well. Is there temporary fix for this issue? some hack?

**Comment by SeanNaren** [comment]
As @ryanleary mentioned, remove the `dim=1` parameter.

@ryanleary good catch, not sure of the best way to handle this. I think a start would be to revert the dim=1 in order to run on pytorch stable!

**Comment by jinserk** [comment]
Sorry for making trouble. I guess we can avoid this issue for try and except clause until the dim option becomes more popular.

**Comment by SeanNaren** [comment]
Just `th test.lua` this script https://github.com/baidu-research/warp-ctc/blob/master/torch_binding/tests/test.lua

**Comment by SeanNaren** [comment]
Reverted this for now, but its hella annoying having this message...

**Comment by ryanleary** [comment]
Could add a torch version check in the model definition, and then set the params using `**kwargs` or something? /shrug

**Comment by SeanNaren** [comment]
@ryanleary thats a good idea, I might do that because it gets unwieldy when doing evaluations

**Comment by ryanleary** [comment]
Now using the `dim` arg
